go.mod

```diff
@@ -11,10 +11,10 @@ require (
 	github.com/buildpacks/lifecycle v0.10.2
 	github.com/containerd/containerd v1.5.9 // indirect
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-openapi/spec v0.20.3
+	github.com/go-openapi/spec v0.20.4
 	github.com/google/go-cmp v0.5.7
-	github.com/google/go-containerregistry v0.8.0
-	github.com/google/go-containerregistry/pkg/authn/k8schain v0.0.0-20210610160139-c086c7f16d4e
+	github.com/google/go-containerregistry v0.8.1-0.20220125170349-50dfc2733d10
```

Comment:
Can we use a released version of ggcr?

Reply:
Will the replace directive on line 46 take care of that, perhaps? If I recall correctly the project does not build with this version but it is a transitive dependency of the version bump.